### PR TITLE
brotli: bump version

### PR DESCRIPTION
**Pull request by kenmays**
Tested on Haiku R1B4 x64.
- Passed all unit tests.
- brotli 1.1.0

### CODE REVIEW

File: app-arch/brotli/brotli-1.0.9.recipe [hunk 1]
```diff
@@ -120,9 +107,6 @@ INSTALL()
 	prepareInstalledDevelLibs libbrotli{common,dec,enc}
 	fixPkgconfig
 
-	packageEntries devel \
-		"$manDir/man3" \
-		"$developDir"
```
Comment (jmairboeck) [inline app-arch/brotli/brotli-1.0.9.recipe:125]: This should be replaced with `rm -r` instead (see #10004).

**Review by jmairboeck — CHANGES_REQUESTED** [review]

**Comment by korli** [comment]
What's the purpose of keeping the recipe for 1.0.9?

**Comment by kenmays** [comment]
@korli - Fallback for current packages until we migrate them.

**Comment by Begasus** [comment]
Needs to be checked if SONAME change(?) doesn't brake existing installs.
```
~> inrecipe lib:libbrotlidec
/Opslag/haikuports/app-arch/brotli/brotli-1.0.9.recipe:32:      lib:libbrotlidec$secondaryArchSuffix = $libVersionCompat
/Opslag/haikuports/dev-qt/qt6-base/qt6_base-6.6.0.recipe:43:    lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/media-gfx/fontforge/fontforge-20230101.recipe:38:    lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/media-libs/gst_plugins_good/gst_plugins_good-1.22.0.recipe:28:       lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/media-libs/libjxl/libjxl-0.6.1.recipe:45:    lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/media-libs/libjxl/libjxl-0.9.0.recipe:45:    lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/media-libs/woff2/woff2-1.0.2.recipe:26:      lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/net-libs/libsoup/libsoup3-3.2.2.recipe:26:   lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/net-libs/webkit-gtk/webkit_gtk-2.40.0.recipe:39:     lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/net-misc/remmina/remmina-1.4.29.recipe:32:   lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/net-misc/wget2/wget2-2.1.0.recipe:41:        lib:libbrotlidec$secondaryArchSuffix
/Opslag/haikuports/www-client/epiphany/epiphany-43.1.recipe:39: lib:libbrotlidec$secondaryArchSuffix
```

**Comment by kenmays** [comment]
To minimize API/ABI crossover with callbacks, I can help migrate dependencies to the newer libs once merged.

**Comment by korli** [comment]
> @korli - Fallback for current packages until we migrate them.

There is no fallback if the recipe names are identical. Why would there be any packages to migrate?

**Pull request merged**